lib/ably/rest/client.rb

```diff
@@ -167,10 +166,10 @@ class Client
       # @return [Ably::Rest::Client]
       #
       # @example
-      #    # create a new client authenticating with basic auth
+      #    # The Ably API key or token string used to validate the client.
```

Comment:
This seems wrong to me, the comment before was fine and the new one doesn't fit.